onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_api_helper.cc

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/common/inlined_containers_fwd.h"
+#include "core/providers/nnapi/nnapi_builtin/builders/model_builder.h"
+#include "core/providers/nnapi/nnapi_builtin/nnapi_lib/nnapi_implementation.h"
+#include "core/providers/nnapi/nnapi_builtin/nnapi_api_helper.h"
```

Comment:
this should be the first include, per https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes